### PR TITLE
OHRM-625: Initiate buzz plugin

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: symfony/apps/orangehrm/templates/_header.php [hunk 1]
```diff
@@ -37,6 +37,7 @@
         echo javascript_include_tag('jquery/jquery.tipTip.minified.js');
         echo javascript_include_tag('jquery/bootstrap-modal.js');
         echo javascript_include_tag('jquery/jquery.clickoutside.js');
+        echo javascript_include_tag('fontawesome-5.13.0/js/all.min.js');
```
Comment (samanthajayasinghe) [inline symfony/apps/orangehrm/templates/_header.php:40]: We need to think of moving all icons to FA

File: symfony/lib/model/doctrine/orangehrmBuzzPlugin/BuzzNotificationMetadata.class.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
+ * BuzzNotificationMetadata
+ * 
+ * This class has been auto-generated by the Doctrine ORM Framework
+ * 
+ *  
+ * @package    orangehrm
```
Comment (samanthajayasinghe) [inline symfony/lib/model/doctrine/orangehrmBuzzPlugin/BuzzNotificationMetadata.class.php:9]: I know this is generated stuff.. any chance to chnage this template?

File: symfony/plugins/orangehrmBuzzPlugin/config/ws_config.yml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+methods:
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmBuzzPlugin/config/ws_config.yml:1]: do we need this ws-config ?

File: symfony/plugins/orangehrmBuzzPlugin/config/view.yml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+viewBuzzSuccess:
+  layout: chatter
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmBuzzPlugin/config/view.yml:2]: hope we going to move into default layout

File: symfony/plugins/orangehrmBuzzPlugin/lib/dao/BuzzNotificationDao.php [hunk 5]
```diff
@@ -0,0 +1,216 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmBuzzPlugin/lib/dao/BuzzNotificationDao.php:4]: header is wrong

File: symfony/plugins/orangehrmBuzzPlugin/lib/utility/BuzzImageCompressUtility.php [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+                
+                $message = $photo->filename . "({$photo->width} x {$photo->height}) size: " . strlen($photo->photo) . " bytes => ";
+                $photo->width = imagesx($image);
+                $photo->height = imagesy($image);
+                $photo->photo = $imageUtility->getImageDataFromResource($image);
+                $photo->size = strlen($photo->photo);
+                
+                $message .= "reduced to: ({$photo->width} x {$photo->height}) size: " . strlen($photo->photo);
+                
+                //echo($message . "\n");
```
Comment (samanthajayasinghe) [inline symfony/plugins/orangehrmBuzzPlugin/lib/utility/BuzzImageCompressUtility.php:73]: remove this

File: symfony/plugins/orangehrmBuzzPlugin/lib/dao/BuzzNotificationDao.php (hunk 5 above)
Comment (RajithaKumara) [reply symfony/plugins/orangehrmBuzzPlugin/lib/dao/BuzzNotificationDao.php:4]: Oops. But seems fine. Can you check this [https://github.com/orangehrm/orangehrm/blob/master/symfony/plugins/orangehrmPimPlugin/modules/pim/actions/addEmployeeAction.class.php](https://github.com/orangehrm/orangehrm/blob/master/symfony/plugins/orangehrmPimPlugin/modules/pim/actions/addEmployeeAction.class.php)

File: symfony/plugins/orangehrmBuzzPlugin/config/ws_config.yml (hunk 3 above)
Comment (RajithaKumara) [reply symfony/plugins/orangehrmBuzzPlugin/config/ws_config.yml:1]: But removing this plugin will break

File: symfony/lib/model/doctrine/orangehrmBuzzPlugin/BuzzNotificationMetadata.class.php (hunk 2 above)
Comment (RajithaKumara) [reply symfony/lib/model/doctrine/orangehrmBuzzPlugin/BuzzNotificationMetadata.class.php:9]: I'll look into that

**Pull request merged**